rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import babel from "@rollup/plugin-babel";
+import commonjs from "@rollup/plugin-commonjs";
+import json from "@rollup/plugin-json";
+import resolve from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import svgr from "@svgr/rollup";
+import analyze from "rollup-plugin-analyzer";
+import cleaner from "rollup-plugin-cleaner";
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
+
+const plugins = [
+  cleaner({
+    targets: ["./dist/"],
+  }),
+  peerDepsExternal(),
+  replace({
+    "process.env.NODE_ENV": JSON.stringify("production"),
+    preventAssignment: true,
+  }),
+  resolve({
+    browser: true,
+    preferBuiltins: true,
+    extensions: [".js", ".jsx", ".svg"],
+  }),
+  svgr(),
+  commonjs({ include: "node_module/" }),
+  json(),
```

Comment:
What are you using `json()` plugin for?

Reply:
Right now not using it anywhere. 

Same as https://github.com/bigbinary/neeto-hotkeys/pull/9#discussion_r1396720635